assets/i18n/en.json

```diff
@@ -17,7 +17,18 @@
     "I18N_CREATE_YES_PLEASE": "Yes, please!",
     "I18N_DASHBOARD_COLLECTIONS": "Collections",
     "I18N_DASHBOARD_EXPLORATIONS": "Explorations",
+    "I18N_DASHBOARD_EXPLORATIONS_SORT_BY": "Sort By",
+    "I18N_DASHBOARD_EXPLORATIONS_SORT_BY_AVGRATING" : "Average Rating",
+    "I18N_DASHBOARD_EXPLORATIONS_SORT_BY_LASTUPDATED": "Last Updated",
```

Comment:
LASTUPDATED --> LAST_UPDATED

Ditto below. No need to makethingsintooneword -- it's harder to read.